lib/charms/mysql/v0/mysql.py

```diff
@@ -1955,6 +1961,7 @@ def is_instance_in_cluster(self, unit_label: str) -> bool:
                 user=self.server_config_user,
                 password=self.server_config_password,
                 host=self.instance_def(self.server_config_user),
+                exception_as_warning=True,
```

Comment:
Can you please share some explanation here? Tnx!

Reply:
When the method is called on an instance that does not belong to the cluster (initialization, some self healing cases), it throws an noisy error log which is not a failure per se, but a valid output.